app/scripts/lib/transaction/util.test.ts

```diff
@@ -66,12 +67,18 @@ const TRANSACTION_OPTIONS_MOCK: AddTransactionOptions = {
   type: TransactionType.simpleSend,
 };
 
-const DAPP_REQUEST_MOCK = {
-  id: TRANSACTION_OPTIONS_MOCK.actionId,
+const makeDappRequest = () => ({
+  jsonrpc: '2.0' as const,
+  id: TRANSACTION_OPTIONS_MOCK.actionId as string,
   method: 'eth_sendTransaction',
-  origin: TRANSACTION_OPTIONS_MOCK.origin,
-  securityAlertResponse: { test: 'value' },
-};
+  params: [],
+});
+
+const makeRequestContext = () =>
+  new MiddlewareContext<Record<PropertyKey, unknown>>({
+    origin: TRANSACTION_OPTIONS_MOCK.origin as string,
```

Comment:
Nit: Unnecessary cast
```suggestion
    origin: TRANSACTION_OPTIONS_MOCK.origin,
```